source/3dcitydb/docker.rst

```diff
@@ -460,44 +460,108 @@ that may help to increase database performance.
 
-1. Start a 3DCityDB Docker container.
+1. Choose a 3DCityDB image that is suitable for you purpose. You will not be able
+   to change the image version later, as you could easily do when using volumes
+   (the default). Available versions are listed in :ref:`citydb_docker_image_variants`.
+   To update an image with data, it has to be recreated from scrap using the
+   desired/updated base image.
+
+2. Start a 3DCityDB Docker container:
 
   .. code-block:: bash
 
-    docker run -d --name citydb \
+    docker run -d --name citydbTemp \
       -e "PGDATA=/mydata" \
       -e "POSTGRES_PASSWORD=changeMe!" \
       -e "SRID=25832" \
     3dcitydb/3dcitydb-pg:4.1.0-alpine
 
-2. Import data to the container. For this example we are use the
+  .. warning:: The database credentials and settings provided in this step
+    cannot be changed when later on creating containers from this image!
+
+    Note down the database connection credentials (db name, username, password)
+    or you won't be able to access the content later.
+
+2. Import data to the container. For this example we are using the
    `LoD3 Railway dataset <https://github.com/3dcitydb/importer-exporter/raw/
    92e08aa306611ee850e065bb542bb3d60791a54f/resources/samples/
-   Railway%20Scene/Railway_Scene_LoD3.zip>`_.
+   Railway%20Scene/Railway_Scene_LoD3.zip>`_ and the
+   :ref:`3DCityDB Importer/Exporter Docker image<impexp_docker_chapter>`:
 
   .. code-block:: bash
 
     docker run -i -t --rm --name impexp \
-        --link citydb \
+        --link citydbTemp \
         -v /d/temp:/data \
       3dcitydb/impexp:edge-alpine import \
-        -H citydb \
+        -H citydbTemp \
         -d postgres \
         -u postgres \
         -p changeMe! \
-        /data/building.gml
+        /data/Railway_Scene_LoD3.zip
+
+3. Stop the running 3DCityDB container and commit it to an image:
+
+  .. code-block:: bash
 
-3. Stop running 3DCityDB container commit 3DCityDB container to an image
+    docker stop citydbTemp
+    docker commit citydbTemp 3dcitydb/3dcitydb-pg:4.1.0-alpine-railwayScene_LoD3
+
+4. Remove the 3DCityDB container:
 
   .. code-block:: bash
 
-    docker stop citydb
-    docker commit citydb 3dcitydb/3dcitydb-pg:4.1.0-alpine-railwayScene_LoD3
+    docker rm -f -v citydbTemp
+
+We have now created a 3DCityDB image that contains data that can e.g. be pushed to a
+Docker registry or exported ar TAR.
+When creating containers from this image, it is not required to specify any configuration
+parameter as you usually would, when creating a fresh 3DCityDB container.
+
+.. code-block:: bash
+
+  docker run --name cdbWithData --rm -p 5432:5432 \
+    3dcitydb/3dcitydb-pg:4.1.0-alpine-railwayScene_LoD3
+
+To connect to the database, use the credentials you set in step 2. The following example
+lists the tables of the DB running in the container using ``psql``.
+
+.. code-block:: console
+
+  $ PGPASSWORD=postgres psql -h localhost -p 5432 -U postgres -d postgres -c '\dt'
```

Comment:
It seems that the first part of the command is missing?

Reply:
I think, it's fine like this. 
The $ indicates a command for the *console* syntax highlighting. In other rST/Sphinx docus I use it a lot in combination with this: https://sphinx-copybutton.readthedocs.io/en/latest/
It will then copy only the command and not the command output in the example. The copybutton might be helpful for us as well. I never tried it on docs hosted at readthedocs.io though. I'll add a PR for testing.

`PGPASSWORD=postgres psql -h localhost -p 5432 -U postgres -d postgres -c '\dt'` 
This sets the PGPASSWORD var for psql just for this single command.
I changed it anyway for better overview and an example with counting features.